interface/globals.php

```diff
@@ -403,10 +403,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 
 // 1 = send email message to given id for Emergency Login user activation,
 // else 0.
-$GLOBALS['Emergency_Login_email'] = $GLOBALS['Emergency_Login_email_id'] ? 1 : 0;
+$GLOBALS['Emergency_Login_email'] = (
+  array_key_exists('Emergency_Login_email_id', $GLOBALS)
+  && $GLOBALS['Emergency_Login_email_id']
+)
+  ? 1
+  : 0;
```

Comment:
This is fine but PHP offers an easier way:
$GLOBALS['Emergency_Login_email'] = empty($GLOBALS['Emergency_Login_email_id']) ? 0 : 1;